src/main/scala/beam/competition/run/CompetitionHelper.scala

```diff
@@ -325,7 +325,7 @@ trait CompetitionHelper extends BeamHelper {
     // uploadCompetitionResultsToS3(competitionServices)
 
     runStateMonitor.setState(RunStateTemplates.SUCCESS)
-    runStateMonitor.syncStateWithRedis(forceUpdate = true)
+    //runStateMonitor.syncStateWithRedis(forceUpdate = true)
```

Comment:
This was removed in other parts of BISTRO -you'd have to re-add from BISTRO first